src/RTCPeerConnection.js

```diff
@@ -94,13 +97,38 @@ export default class RTCPeerConnection extends defineCustomEventTarget(...PEER_C
         WebRTCModule.peerConnectionRemoveStream(stream._reactTag, this._peerConnectionId);
     }
 
+    addTransceiver(source: 'audio' |'video' | MediaStreamTrack, init) {
+        return new Promise((resolve, reject) => {
+            let src;
+            if (source === 'audio') {
+                src = { type: 'audio' };
+            } else if (source === 'video') {
+                src = { type: 'video' };
+            } else {
+                src = { trackId: source.id };
+            }
+
+            WebRTCModule.peerConnectionAddTransceiver(this._peerConnectionId, {...src, init: { ...init } }, (successful, data) => {
+                if (successful) {
+                    this._mergeState(data.state);
+                    console.log('NATIVE added transceiever OK')
+                    resolve(this._transceivers.find((v) => v.id === data.id));
+                } else {
+                    console.log('NATIVE ERROR', data);
+                    reject(data);
+                }
+            });
+        });
+    };
+
     createOffer(options) {
         return new Promise((resolve, reject) => {
             WebRTCModule.peerConnectionCreateOffer(
                 this._peerConnectionId,
                 RTCUtil.normalizeOfferAnswerOptions(options),
                 (successful, data) => {
                     if (successful) {
+                        this._mergeState(data.state);
                         resolve(data);
```

Comment:
```suggestion
                        resolve(data.session);
```

Native is now returning a dictionary containing `state` and `session` (the actual sdp). Need to return the `session` value directly to conform to web api spec which returns the `RTCSessionDescription` directly.